cmd/kube-proxy/app/server.go

```diff
@@ -136,10 +138,19 @@ func NewProxyServerDefault(config *options.ProxyServerConfig) (*ProxyServer, err
 		protocol = utiliptables.ProtocolIpv6
 	}
 
+	var netshInterface utilnetsh.Interface
+	var iptInterface utiliptables.Interface
+	var dbus utildbus.Interface
+
 	// Create a iptables utils.
 	execer := exec.New()
-	dbus := utildbus.New()
-	iptInterface := utiliptables.New(execer, dbus, protocol)
+
+	if runtime.GOOS == "windows" {
```

Comment:
@thockin right now, there's _only_ three of such checks and we can't see a quick and clear way for us to add this in time to have it in 1.5. Again, and as above, we'll have this in consideration for the next release.
Also, this may even go away as Windows Containers evolve.